From c81118e16f8b292b1412e814982fa7d19f534309 Mon Sep 17 00:00:00 2001
From: Tian Yang <tiany@codeaurora.org>
Date: Mon, 1 Feb 2021 17:05:11 -0800
Subject: [PATCH] skbuff_debug: Fix double activation error

Remove the init and activate in __build_skb_around
since it was already finished in __build_skb.

Signed-off-by: Tian Yang <tiany@codeaurora.org>
Change-Id: I4b131dc6e32bb9e8597378f633637627656a46d6
---
 net/core/skbuff.c | 1 -
 1 file changed, 1 deletion(-)

--- a/net/core/skbuff.c
+++ b/net/core/skbuff.c
@@ -215,7 +215,6 @@ static void __build_skb_around(struct sk
 	shinfo = skb_shinfo(skb);
 	memset(shinfo, 0, offsetof(struct skb_shared_info, dataref));
 	atomic_set(&shinfo->dataref, 1);
-	skbuff_debugobj_init_and_activate(skb);
 
 	skb_set_kcov_handle(skb, kcov_common_handle());
 }
